core/azure-core-test/pom.xml

```diff
@@ -5,14 +5,14 @@
   <parent>
     <groupId>com.azure</groupId>
```

Comment:
When we release the azure-core package we should avoid releasing the azure-core-test and other packages. We should only release the ones are libraries depend on which I believe is just azure-core and azure-core-amqp.